sip/sip-001-burn-election.md

```diff
@@ -210,50 +225,73 @@ block-withholding attacks, including selfish mining.  It does not prevent these
-over a distribution of chain tips are possible only because all peers have 
-knowledge of all existing chain tips and the burns behind them.
+* **Ancilliary proofs to hedge bets**:  Because anyone can produce a tunable proof 
+in favor of any chain tip, leaders can hedge their bets on their preferred chain tips by distributing
+their proofs across _all_ competing chain tips.  Both fair mining pools and generating
+proofs over a distribution of chain tips are possible only because all peers have 
+knowledge of all existing chain tips and the proofs behind them.
 
 # Leader Election
 
 The Stacks blockchain makes progress by selecting successive leaders to produce
-blocks.  It does so via burn mining, whereby would-be leaders submit their candidacies
-to be leaders by burning an existing cryptocurrency.  A leader is selected to produce a
-block based on two things:
+blocks.  It does so by having would-be leaders submit their candidacies
+by both burning an existing cryptocurrency, and submitting a (distinct) proof
+of work.  A leader is selected to produce a block based on two things:
 
-* the amount of cryptocurrency burned relative to the other candidates
+* the amount of cryptocurrency burned and energy expended relative to the other candidates
 * an unbiased source of randomness
 
 A new leader is selected whenever the burn chain produces a new block -- the
 arrival of a burn chain block triggers a leader election, and terminates the
 current leader's tenure.
 
-The basic structure for leader election through burn-mining is that
+The basic structure for leader election through tunable proofs is that
 for some Stacks block _N_, the leader is selected via some function of
-that leader's total cryptocurrency burnt in a previous block _N'_ on the
+that leader's proof-of-work and total cryptocurrency burnt in a previous block _N'_ on the
 underlying burn chain.  In such a system, if a candidate _Alice_ wishes to be a leader of a
 Stacks block, she issues a burn transaction in the underlying burn
-chain. The network then uses cryptographic sortition to choose a
-leader in a verifiably random process, weighted by burn amounts in prior blocks.
-The block in which this burn transaction is broadcasted is known
-as the "election block" for Stacks block _N_.
+chain which both destroys some cryptocurrency and encodes a proof-of-work.
+The network then uses cryptographic sortition to choose a
+leader in a verifiably random process, weighted by the sums of the burn amounts and 
+proofs-of-work in prior blocks.  The block in which this burn transaction is
+broadcasted is known as the "election block" for Stacks block _N_.
 
 Anyone can submit their candidacy as a leader by issuing a burn transaction on
 the underlying burn chain, and have a non-zero chance of being selected by the
 network as the leader of a future block.
 
+The proof-of-work and the proof-of-burn are converted into a combined tunable
+proof score by the following process:
```

Comment:
This process seems like it's trying to do too much to combine PoW and PoB into comparable units. Why not just compare ranks?

Thinking about how this is implemented with sortition, it would look something like:

```
W_ib = scaled burn amount for candidate i = (burn_i / burn_total)
W_iw = scaled work amount for candidate i = (work_i / work_total)

t := "proof-of-work" tuning factor
T_i = tuned score for candidate i = (1-t)(W_ib) + t(W_iw)
```

And then using `T_i` to claim ranges over the sortition space. This way, the tunable proofs don't need to try and compare the amount of work going into one component versus the other -- it just lets the competitive aspect of mining balance that.

Reply:
That makes more sense.  I was getting hung up when I was writing this section in finding a way to make sure the units for `W_ib` and `W_iw` were the same, but in this particular application that doesn't matter.